modules/ROOT/partials/configuration/autocorrect_service_url.adoc

```diff
@@ -0,0 +1,21 @@
+[[autocorrect_service_url]]
+
+== `+autocorrect_service_url+`
+
+This option specifies the URL of the server-side `+ephox-spelling+` service. For instructions on how to set up a Spellchecking server-side component, see: the xref:introduction-to-premium-selfhosted-services.adoc[server-side component installation guide].
```

Comment:
This needs an admonition below this about not being needed for Tiny Cloud like all other service url options. See https://www.tiny.cloud/docs/tinymce/6/export/#export_image_proxy_service_url for an example.